python/shotgun_model/simple_shotgun_model.py

```diff
@@ -70,6 +70,7 @@ def load_data(
                   is handling, allowing a user to for example show the twenty most recent notes or
                   similar.
         :param columns: List of Shotgun fields to use to populate the model columns
+        :param editable_columns: A subset of ``columns`` that will be editable in views that use this model.
         :param additional_filter_presets: List of Shotgun filter presets to apply, e.g.
```

Comment:
It's still a mystery to me as well. I think this is all server side now and undocumented. I'll add to Wed meeting agenda for explanation/discussion.